### PR TITLE
Adding affiliate definition

### PR DESCRIPTION
**Pull request by Ainali**
Solves #496 

-----
[View rendered glossary/affiliate-definition.md](https://github.com/publiccodenet/about/blob/glossary-affiliate/glossary/affiliate-definition.md)
[View rendered glossary/index.md](https://github.com/publiccodenet/about/blob/glossary-affiliate/glossary/index.md)

### CODE REVIEW

**Comment by bvhme** [comment]
Great pull request. Affiliates should also be non-profit.

**Comment by Ainali** [comment]
> Great pull request. Affiliates should also be non-profit.

I was unsure about that so thanks for the input. I have updated the definition now.

File: glossary/affiliate-definition.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+type: Resource
+---
+
+# Affiliate
+
+Non-profit, non-governmental organization that has signed a letter of interest of collaborating with the Foundation for Public Code.
```
Comment (ElenaFdR) [inline glossary/affiliate-definition.md]:
```suggestion
Non-profit, non-governmental organization that has signed a letter of interest in collaborating with the Foundation for Public Code.
```

File: glossary/affiliate-definition.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+type: Resource
+---
+
+# Affiliate
+
+Non-profit, non-governmental organization that has signed a letter of interest in collaborating with the Foundation for Public Code.
```
Comment (bvhme) [inline glossary/affiliate-definition.md]: I would suggest removing `non-govermental` as some might be, e.g. we have a relationship with OASC, a network of cities. The requirement for the kind of relationship can also be looser, with OS2 we have a _memorandum of understanding_ and with OW2 an _affiliate contract_.

```suggestion
A non-profit organization that has an aligned mission with us and we have formal relations with.
```

Comment (bvhme) [reply glossary/affiliate-definition.md]: Ah yes, and the 'aligned with our mission' seems important :)

**Pull request merged**